### ISSUE TITLE
[Bug] 22w47a - Blocks can be placed so they clip on child entities

### ISSUE THREAD

**Issue by Dwarf-LordPangolin** [tsunamayo/Starship-EVO#5011]
### Short Description of the issue:
Blocks on child entities can be placed so they clip into other blocks on child entities. The issue does not appear to affect the parent entity.

### Steps to reproduce the issue:
I'm having trouble duplicating this bug on other entities, so I'm including the specific blueprint it's occurring on. This bug might be related to the copy-paste function; the child entity in question is a ship's port wing that was pasted after copying it from the starboard side, which does not have this issue.

### Environment
- Starship EVO version: 22w47a
- Graphics Card: AMD Radeon RX 5600 XT

### Additional information:
On child entity:
![20221203021332_1](https://user-images.githubusercontent.com/33041786/205431584-ecac869c-ee0e-4a32-89b9-d9f7fc186633.jpg)

![20221203021335_1](https://user-images.githubusercontent.com/33041786/205431593-8f00aaa6-4823-463a-9bb2-93e46b123d61.jpg)

Versus on parent entity:
![20221203021607_1](https://user-images.githubusercontent.com/33041786/205431601-554f92c5-c022-4217-81fb-a92019dcec77.jpg)

[Player.log](https://github.com/tsunamayo/Starship-EVO/files/10145972/Player.log)

[AAM Transport Prototype.zip](https://github.com/tsunamayo/Starship-EVO/files/10145973/AAM.Transport.Prototype.zip)

**Comment by Asnro** [comment]
Did you try restarting the game? I've noticed copy-pasted child entities lack collision until after a restart, so I wonder if this is related.

**Comment by Dwarf-LordPangolin** [comment]
Good catch. Restarting does indeed cause it to stop happening, so I think you're correct: it's a collision issue with bits copy-pasted onto child entities until the game's restarted.
